src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -52,21 +61,28 @@ const AuctionsApp: React.FC<AuctionsAppProps> = props => {
         </Column>
       </GridColumns>
 
-      {user && (
-        <>
-          <Spacer py={2} />
-          <Join separator={<Spacer my={[2, 6]} />}>
-            <Box>
-              {/* @ts-expect-error STRICT_NULL_CHECK */}
-              <MyBidsFragmentContainer me={viewer.me} />
-            </Box>
+      <Spacer mt={4} />
+      <Tabs>
+        {user && (
+          <>
```

Comment:
Don't think this `<> is necessary since `<Tab>` encloses the whole section

Reply:
Removed!